### PR TITLE
#FP-19 Multi stopover update

### PR DESCRIPTION
**Pull request by TobyChurches**
Both Standard and advanced stopover working - However, advanced will not work properly until the database is properly developed (all flights currently have no free seats, thus the search is always restricted on the seat capacity)

### CODE REVIEW

File: src/main/java/com/FlightPub/Controllers/IndexController.java [hunk 1]
```diff
@@ -219,19 +219,20 @@ public String runSearch(@ModelAttribute BasicSearch search, Model model, HttpSes
...
 
     @PostMapping("/advancedSearch")
-    public String runAdvancedSearch(@ModelAttribute BasicSearch search, Model model, HttpSession session)
-    {
+    public String runAdvancedSearch(@ModelAttribute BasicSearch search, Model model, HttpSession session) {
         model = addDateAndTimeToModel(model);
         List<Flight> flights;
-        List<SingleStopOver> flights1Stop = null;
-        List<MultiStopOver> flights2Stop = null;
+        List<StopOver> flights1Stop = null;
```
Comment (WillPaterson) [inline src/main/java/com/FlightPub/Controllers/IndexController.java]: Would is it easier to convert this to an array and just reference the indexes?

Comment (TobyChurches) [reply src/main/java/com/FlightPub/Controllers/IndexController.java]: Good Call

**Review by WillPaterson — APPROVED** [review]

File: src/main/java/com/FlightPub/Controllers/IndexController.java (hunk 1 above)
Comment (TobyChurches) [reply src/main/java/com/FlightPub/Controllers/IndexController.java]: Should be fixed now @WillPaterson

Comment (WillPaterson) [reply src/main/java/com/FlightPub/Controllers/IndexController.java]: Yeah, looks good 👍

**Review by Reason0x6 — APPROVED** [review]

**Pull request merged**